starters/next13-zustand-bulma/README.md

```diff
@@ -1,38 +1,117 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+# next13-zustand-bulma Starter Kit
 
-## Getting Started
+This starter kit features [**Next.js 13+**](https://beta.nextjs.org/docs) (with the `appDir` and server components), [**Zustand**](https://zustand-demo.pmnd.rs/) for client-side state management, and [**Bulma**](https://bulma.io/) for styling.
 
-First, run the development server:
+##### How to use this Starter Kit:
 
 ```bash
-npm run dev
-# or
-yarn dev
-# or
-pnpm dev
+npm create @this-dot/starter -- --kit next13-zustand-bulma
 ```
 
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+<sub>(use any of `npm` / `yarn` / `pnpm`)</sub>
+
+_[(...or use one of our alternative setup instructions)](SETUP.md)_
+
+<details>
+<summary>&nbsp;&nbsp;<b>Table of Contents</b></summary>
+
+- [next13-zustand-bulma starter kit](#next13-zustand-bulma-starter-kit)
+  - [Tech Stack](#tech-stack)
+    - [Included Tooling](#included-tooling)
+  - [Getting Started](#getting-started)
+  - [Available Commands](#available-commands)
+  - [Project Details](#project-details)
+    - [Kit Organization / Architecture](#kit-organization--architecture)
+    - [Styling and Theme](#styling-and-theme)
+    - [Font Loading](#font-loading)
+    - [Colocation in directories](#colocation-in-directories)
+  - [Demo Implementation](#demo-implementation)
+  - [Deployment](#deployment)
+
+</details>
+
+## Tech Stack
+
+- [Next.js v13.x](https://nextjs.org)
+- [React v18.x](https://reactjs.org)
+- [Zustand v4.x](https://react-query.tanstack.com/)
```

Comment:
It looks like we are using the wrong link here

Reply:
What a fantastic review, all good catches with an astute eye.

fixed in 2d02ec6